velox/exec/tests/PlanNodeToStringTest.cpp

```diff
@@ -198,7 +198,7 @@ TEST_F(PlanNodeToStringTest, aggregation) {
 
   ASSERT_EQ("-- Aggregation[1]\n", plan->toString());
   ASSERT_EQ(
-      "-- Aggregation[1][PARTIAL a := sum(ROW[\"c0\"]), b := avg(ROW[\"c1\"]), c := min(ROW[\"c2\"]), a3 := count(ROW[\"c1\"]) distinct] "
+      "-- Aggregation[1][ALLOW FLUSH PARTIAL a := sum(ROW[\"c0\"]), PARTIAL b := avg(ROW[\"c1\"]), PARTIAL c := min(ROW[\"c2\"]), PARTIAL a3 := count(ROW[\"c1\"]) distinct] "
```

Comment:
This makes the plan harder to read. Perhaps, when all aggs use same step we could keep the original format and use new format only when there is a mix of steps.

Reply:
fixe in https://github.com/facebookincubator/velox/pull/13024